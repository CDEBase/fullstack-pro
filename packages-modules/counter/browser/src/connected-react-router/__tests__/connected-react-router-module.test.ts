import Module from '../module';

import 'jest';

describe('connector modules', () => {

    it('module configuredRoutes', () => {
        const configuredRoutes = Module.getConfiguredRoutes();

        expect(configuredRoutes).toMatchSnapshot();
    });

    it('module routes', () => {
        const routes = Module.getRoutes();

        expect(routes).toMatchSnapshot();
    });
});
