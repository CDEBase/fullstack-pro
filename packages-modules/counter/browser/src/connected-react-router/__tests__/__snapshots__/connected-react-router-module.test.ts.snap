// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`connector modules module configuredRoutes 1`] = `
Array [
  Object {
    "component": [Function],
    "exact": true,
    "path": "/connected-react-router",
    "routes": Array [
      Object {
        "component": [Function],
        "exact": true,
        "path": "/connected-react-router/hello",
      },
      Object {
        "component": Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Counter)",
          "type": [Function],
        },
        "exact": true,
        "path": "/connected-react-router/counter",
      },
    ],
  },
]
`;

exports[`connector modules module routes 1`] = `
<Switch>
  <Route
    exact={true}
    path="/connected-react-router"
    render={[Function]}
  />
</Switch>
`;
